### PR TITLE
New job for calling CGI added (#7)

### PR DESCRIPTION
**Pull request by saravanpa-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/6246

*Description of changes:*
Added new job for FEEDBACK and ACK jobs

Test cases are missing as of now. The challenge is since the files are getting moved around , it changes files in the local file system as well ;which makes it a bad test. I might create a tech debt ticket to address this incase if i cant do it in my next PR

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

File: jobs/ftp-poller/tasks/cas_poller_ftp.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Copyright © 2019 Province of British Columbia
```
Comment (saravanpa-aot) [inline jobs/ftp-poller/tasks/cas_poller_ftp.py:1]: this file got renamed.contents are same

File: jobs/ftp-poller/invoke_jobs.py [hunk 2]
```diff
@@ -64,16 +64,16 @@ def shell_context():
...
     application = create_app()
 
     application.app_context().push()
     if job_name == 'FTP_POLLER':
-        PollFtpTask.poll_ftp()
+        CASPollerFtpTask.poll_ftp()
         application.logger.info(f'<<<< Completed Polling FTP >>>>')
     else:
-        PollFtpTask.poll_ftp()
+        CASPollerFtpTask.poll_ftp()
```
Comment (sumesh-aot) [inline jobs/ftp-poller/invoke_jobs.py:76]: Is this correct, or should it be cgi poller ftp ?

**Review by sumesh-aot — APPROVED** [review]
Check my comments

File: jobs/ftp-poller/invoke_jobs.py (hunk 2 above)
Comment (saravanpa-aot) [reply jobs/ftp-poller/invoke_jobs.py]: Corrected it.Thanks for catching it

**Pull request merged**